examples/lib/defaultHtmlStepUi.js

```diff
@@ -125,23 +130,24 @@ function DefaultHtmlStepUi(_sequencer, options) {
                          </div>";
         step.ui.querySelector("div.details").appendChild(div);
       }
-
-      $(step.ui.querySelector("div.details")).append(
+      $(step.ui.querySelector("div.panel-footer")).append(
```

Comment:
Again, try not to append too many things, it makes it difficult to read the code and navigate.

Reply:
I need to append because of place in which it needs to be.

Reply:
:+1: